src/main/java/org/carlspring/strongbox/janusgraph/cassandra/CassandraEmbeddedConfig.java

```diff
@@ -36,4 +39,13 @@ public CassandraDaemon activateCassandraDaemon(CassandraEmbeddedProperties prope
         return cassandraDaemon;
     }
 
+    @PreDestroy
+    public void destroy()
```

Comment:
@anki2189 I suggest to make the `CassandraShutdown` class, wihich will have the `CassandraDaemon` instance reference. This bean shoud depends on `CassandraDaemon` and have `shutdown` method with following logic:
```
cassandraDaemon.deactivate();
StorageService.instance.drain();
```
This method should be ceclared as `destroyMethod` of bean.

Reply:
JanusGraph bean should depends on `CassandraShutdown` as well, instead of `CassandraDaemon` directly.

Reply:
@sbespalov Changes done as suggested.